instrumentation/redis/lib/opentelemetry/instrumentation/redis/patches/client.rb

```diff
@@ -10,6 +10,8 @@ module Redis
       module Patches
         # Module to prepend to Redis::Client for instrumentation
         module Client
+          SET_VALUE_SIZE_COMMANDS = %i[set]
+          RETRIEVED_VALUE_SIZE_COMMANDS = %i[get mget]
```

Comment:
If this set gets large, we should consider using a `Set`. Given the small size and use of symbols, the current implementation is likely more efficient.